### ISSUE TITLE
Unable to import './files.html' after upgrading from 1.1.0 to 1.2.0

### ISSUE THREAD

**Issue by doberkofler** [webpack-contrib/html-loader#313]
- Operating System: macOS
- Node Version: 14.8.0
- NPM Version: 6.14.7
- webpack Version: 4.44.1
- html-loader Version: 1.2.0

### Expected Behavior

`import MARKUP from './files.html';` failing after upgrading from 1.1.0 to 1.2.0

### Actual Behavior

```
ERROR in ./src/framework/dropzone/files.html
Module build failed (from ./node_modules/html-loader/dist/cjs.js):
URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at normalizeUrl (/Users/doberkofler/MyDev/ljs_app/trunk/periscope/node_modules/html-loader/dist/utils.js:358:10)
    at process (/Users/doberkofler/MyDev/ljs_app/trunk/periscope/node_modules/html-loader/dist/plugins/source-plugin.js:207:45)
    at Object.process (/Users/doberkofler/MyDev/ljs_app/trunk/periscope/node_modules/html-loader/dist/utils.js:538:19)
    at Object.loader (/Users/doberkofler/MyDev/ljs_app/trunk/periscope/node_modules/html-loader/dist/index.js:58:41)
```

### Code

```js
import MARKUP from './files.html';
```

**Comment by evilebottnawi** [comment]
Please create reproducible test repo

**Comment by evilebottnawi** [comment]
Maybe you are real have bad URL, I can't help without source code of `files.html`

**Comment by doberkofler** [comment]
[files.html.zip](https://github.com/webpack-contrib/html-loader/files/5091739/files.html.zip)

**Comment by evilebottnawi** [comment]
WIP, in the next major release, it will be an error, you will need to use the `urlFilter` option to exclude non standard `src`/`srcset`

**Comment by doberkofler** [comment]
I do not understand your comment:
1) It already is an error in 1.2.0
2) what am I supposed to do with an ‘urlFilter’?

**Comment by evilebottnawi** [comment]
> It already is an error in 1.2.0

It was fixed in `1.2.1` https://github.com/webpack-contrib/html-loader/pull/314, because it is the regression

> what am I supposed to do with an ‘urlFilter’?

In the next major release (`html-loader@2`) it will output the error, because it is not URL, so you need to use `urlFilter` to keep them in template without errors using them:
```js
urlFilter: (attribute, value, resourcePath) => {
  if (/^<%= .+ %>$/.test(value)) {
    return false;
  }

  return true;
},
```

**Comment by doberkofler** [comment]
Thank you!
